html/README.md

```diff
@@ -5,3 +5,15 @@ This directory contains tests for [HTML](https://html.spec.whatwg.org/multipage/
 Sub-directory names should be based on the URL of the corresponding part of the
 multipage-version specification. For example, The URL of
 "8.3 Base64 utility methods" is [https://html.spec.whatwg.org/multipage/webappapis.html#atob](https://html.spec.whatwg.org/multipage/webappapis.html#atob). So the directory in WPT is [webappsapis/atob/](webappsapis/atob).
```

Comment:
Fix this link to webappapis/atob while here?

Reply:
Oh, I didn't notice it was broken, will fix.